docs/web-console-guide_5/master.adoc

```diff
@@ -90,7 +153,35 @@ include::topics_5/web-report-static.adoc[leveloffset=+2]
 ////
 
 // Using Custom Rules
-include::topics_5/web-custom-rules.adoc[leveloffset=+1]
+//include::topics_5/web-custom-rules.adoc[leveloffset=+1]
+
+== Using global rules and labels
```

Comment:
I see no point in combining global rules and labels.

Break them into separate sections and promote each heading.